plugins/GSdx/Renderers/DX11/GSRendererDX11.cpp

```diff
@@ -228,14 +228,12 @@ void GSRendererDX11::EmulateTextureShuffleAndFbmask()
 			break;
 		case ACC_BLEND_MEDIUM_D3D11:
 			// Enable Fbmask emulation excluding triangle class because it is quite slow.
-			// Exclude 0x80000000 because Genji needs sw blending, otherwise it breaks some effects.
-			enable_fbmask_emulation = ((m_vt.m_primclass != GS_TRIANGLE_CLASS) && (m_context->FRAME.FBMSK != 0x80000000));
+			enable_fbmask_emulation = (m_vt.m_primclass != GS_TRIANGLE_CLASS);
 			break;
 		case ACC_BLEND_BASIC_D3D11:
 			// Enable Fbmask emulation excluding triangle class because it is quite slow.
-			// Exclude 0x80000000 because Genji needs sw blending, otherwise it breaks some effects.
 			// Also exclude fbmask emulation on texture shuffle just in case, it is probably safe tho.
-			enable_fbmask_emulation = (!m_texture_shuffle && (m_vt.m_primclass != GS_TRIANGLE_CLASS) && (m_context->FRAME.FBMSK != 0x80000000));
+			enable_fbmask_emulation = (!m_texture_shuffle && (m_vt.m_primclass != GS_TRIANGLE_CLASS));
```

Comment:
Actually I did a bit of testing and it looks like Genji is fixed but only when using High level of blending, didn't test properly before. Basic and Medium will still be broken so it is better to add the check back, comment can be updated tho that we require high level of blending.

Reply:
Hmm, I guess so as all the changes for impossible blends like on Genji will only be enabled on High blending better to leave Medium/Basic alone. The FBMask = 0x80000000 for check is quite hacky though. I think it works because twice the alpha is incorrectly stored in the framebuffer (0xff instead of 0x7f I think) but then when its read back for the impossible blend only half its value is used again incorrectly so it cancels out.